packages/daf-rest/api/daf-rest.api.md

```diff
@@ -25,7 +25,7 @@ export class AgentRestClient implements IAgentPlugin {
     }
 
 // @public (undocumented)
-export const getOpenApiSchema: (agent: IAgent, basePath: string, exposedMethods: Array<string>) => OpenAPIV3.Document;
+export const getOpenApiSchema: (agent: IAgent, basePath: string, exposedMethods: Array<string>, name?: string | undefined, version?: string | undefined) => OpenAPIV3.Document;

```

Comment:
Just curious here, do you need to explicitly define the params as `string | undefined` if they already have a question mark?

Reply:
This generated by `api-exctractor`